### PR TITLE
Added method for 869B - The Eternal Immortality

### PR DESCRIPTION
**Pull request by kenduanemabanglo**

### CODE REVIEW

File: Solutions.java [hunk 1]
```diff
@@ -1,3 +1,26 @@
 public class Solutions {
 	
+    // The Eternal Immortality - 869B
+    public static void TheEternalImmortality(long a, long b) throws Exception{
+
+        long re=a+1;
+        if(re>b)
```
Comment (julianalonzo) [inline Solutions.java]: Please put brackets on the if statement {}

Comment (julianalonzo) [inline Solutions.java]: Please put {} after if statement

File: Solutions.java [hunk 2]
```diff
@@ -1,3 +1,26 @@
...
+            System.out.println(1);
+        else
+        {
```
Comment (julianalonzo) [inline Solutions.java]: Move this so that it is beside the else statement

File: Solutions.java [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+            long h=1;
+            while(3>2)
+            {
```
Comment (julianalonzo) [inline Solutions.java]: Move this so that it is beside the while statement

File: Solutions.java [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+                h*=re;
+                h%=10;
+                if(h==0)
```
Comment (julianalonzo) [inline Solutions.java]: Please add brackets after if {}

File: Solutions.java [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+                    break;
+                else if(re==b)
```
Comment (julianalonzo) [inline Solutions.java]: Please add brackets after else if {}

File: Solutions.java (hunk 1 above)
Comment (kenduanemabanglo) [reply Solutions.java]: Done!

File: Solutions.java (hunk 2 above)
Comment (kenduanemabanglo) [reply Solutions.java]: Done!

File: Solutions.java (hunk 3 above)
Comment (kenduanemabanglo) [reply Solutions.java]: Done!

File: Solutions.java (hunk 4 above)
Comment (kenduanemabanglo) [reply Solutions.java]: Done!

File: Solutions.java (hunk 5 above)
Comment (kenduanemabanglo) [reply Solutions.java]: Done!

File: Solutions.java (hunk 1 above)
Comment (julianalonzo) [reply Solutions.java]: Petmalodi!

File: Solutions.java (hunk 2 above)
Comment (julianalonzo) [reply Solutions.java]: Petmalodi!

File: Solutions.java (hunk 3 above)
Comment (julianalonzo) [reply Solutions.java]: Petmalodi!

File: Solutions.java (hunk 4 above)
Comment (julianalonzo) [reply Solutions.java]: Petmalodi!

File: Solutions.java (hunk 5 above)
Comment (julianalonzo) [reply Solutions.java]: Petmalodi!

**Pull request merged**